conventions/admin.py

```diff
@@ -51,8 +51,9 @@ class ConventionAdmin(admin.ModelAdmin):
         "motif_denonciation",
     )
     list_select_related = (
-        "programme",
-        "lot",
+        "programme__bailleur",
+        "programme__administration",
+        "lot__programme",
```

Comment:
This third one is needee by [view_programme](https://github.com/MTES-MCT/apilos/pull/1051/files#diff-3b427ceaabe5e67534410255aa728af6c531845c54e73caee1f40ee92a633a13R9), str repr of convention.lot